### PR TITLE
KML add position type support

### PR DESCRIPTION
**Pull request by Xierumeng**
Add new functions to support position type.

### CODE REVIEW

File: tests/unit/test_kml_conversion.py [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+
+    assert actual_kml_file_path.exists()
+    assert actual_kml_file_path.suffix == KML_SUFFIX
+
+    assert actual_kml_file_path.read_text(encoding="utf-8") == expected_kml_document_path.read_text(
+        encoding="utf-8"
+    )
+
+
+def test_locations_to_kml(
```
Comment (maxlou05) [inline tests/unit/test_kml_conversion.py:174]: Just curious why this one doesn't have "with_save_path" in its name

File: tests/unit/kml_documents/expected_enumerated_positions.kml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<kml xmlns="http://www.opengis.net/kml/2.2" xmlns:gx="http://www.google.com/kml/ext/2.2">
+    <Document id="25">
```
Comment (maxlou05) [inline tests/unit/kml_documents/expected_enumerated_positions.kml:3]: Just curious on how these ids come about? Why is it 25

**Review by maxlou05 — APPROVED** [review]
Conditionally approved

File: tests/unit/test_kml_conversion.py (hunk 1 above)
Comment (Xierumeng) [reply tests/unit/test_kml_conversion.py]: Refactored (added mock).

File: tests/unit/kml_documents/expected_enumerated_positions.kml (hunk 2 above)
Comment (Xierumeng) [reply tests/unit/kml_documents/expected_enumerated_positions.kml]: Did some investigation and it's caused by a global class variable. Added a workaround.

**Pull request merged**